contrib/cirrus/prebuild.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# This script attempts to confirm functional networking and
+# connectivity to essential external servers.  It also verifies
+# some basic environmental expectations and shell-script sanity.
+# It's intended for use early on in the podman CI system, to help
+# prevent wasting time on tests that can't succeed due to some
+# outage, failure, or missed expectation.
+
+source /etc/automation_environment
+source $AUTOMATION_LIB_PATH/common_lib.sh
+
+req_env_vars CI DEST_BRANCH IMAGE_SUFFIX TEST_FLAVOR TEST_ENVIRON \
+             PODBIN_NAME PRIV_NAME DISTRO_NV AUTOMATION_LIB_PATH \
+             SCRIPT_BASE CIRRUS_WORKING_DIR FEDORA_NAME UBUNTU_NAME \
+             VM_IMAGE_NAME
+
+# There's no need to perform further checks on more than one
+# CI platform.  These variables are defined in .cirrus.yml
+# shellcheck disable=SC2154
+if [[ ! "${DISTRO_NV}" =~ ${FEDORA_NAME} ]]; then
+    echo "Skipping additional checks on $DISTRO_NV"
+    exit 0
+fi
+
+# shellcheck disable=SC2154
+$SCRIPT_BASE/cirrus_yaml_test.py
```

Comment:
Looking at the logs, I just see a jumble of output messages with no indication of what is what. I don't think we want `set -x` here, but what would you think of adding: `echo "Running cirrus_yaml_test"`, `echo "running shellcheck"`, so when one of those steps fails it's easy to see what it actually was?

Reply:
Oh sure, didn't realize/remember this was `set -x`.